charts/barman/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: barman
 type: application
 description: Chart for Barman PostgreSQL Backup and Recovery Manager
-version: 0.9.1
```

Comment:
Thanks for your contribution!

As this is an additional feature, please bump to 0.10.0

Reply:
We are also missing the `artifacthub.io/changes` annotations in the Chart.yaml

See https://github.com/adfinis/helm-charts/pull/901 for a good example.

Thank you!